src/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/*
+Starting off with a small set of constants used in multiple places. This should hopefully become something akin to a resource file for localization.
+*/
+
+export class DialogBoxResponses {
+    static readonly inputYes: string = "Yes";
```

Comment:
nit: Can this just be 'Yes'? I think the name of the class makes 'input' redundant

I would also make the name of the file 'DialogBoxResponses' to match the class name (The cosmosdb repo doesn't match the two very often, but it probably should)